README.md

```diff
@@ -22,65 +86,113 @@ For the latter you will need:
 - bundler (see [bundler installation](https://bundler.io/); for version, see [.bundler-version](./.bundler-version))
 - yq (see [yq installation](https://github.com/mikefarah/yq#install))
 
-### Windows support
+#### Windows support
 
 We provide an experimental support for Window 10.
 
 You can run the `win32\install.bat` script to have an unattended installation of Ruby
 and the tools we use. Once installed, you can run the `specs` using `make.bat`
 
-### Running the connector
+### Implementing the connector protocol using the connector framework
```

Comment:
I would describe the available code examples in this section. You can use the examples as boilerplate to speed up your implementation of your connector.